python/triton/compiler.py

```diff
@@ -822,7 +823,10 @@ def kernel_suffix(signature, specialization):
 # ------------------------------------------------------------------------------
 
 
-def make_triton_ir(fn, signature, specialization, constants):
+def build_triton_ir(fn, signature, specialization, constants):
```

Comment:
Why this function starts with "build", while other functions start with "make"?

Reply:
`make` includes optimization. I plan to rename `make_*` to something more explicit (e.g., `make_triton_ir -> ast_to_triton_ir`). I should have tagged this PR as a draft :)

Reply:
Please do and let me know when it's ready for review :)

Reply:
Should be good now if you wanna take a look